### ISSUE TITLE
onStadiumChange

### ISSUE THREAD

**Issue by Leo-MapCreator** [haxball/haxball-issues#370]
In my opinion checksum is missing and it is very important onStadiumChange
onStadiumChange(newStadiumName : string, byPlayer : PlayerObject, newStadiumChecksum : Hexadecimal ) : void

**Comment by Leo-MapCreator** [comment]
Because i can edit the name of the map to exactly the same the API hosts and im able to play the map i want just with that

**Comment by tohey** [comment]
Solving #303 will solve this too, if `getStadium()` returns the whole object.

**Comment by Leo-MapCreator** [comment]
No, what you are talking about relates more to #288 rather than #303
